rust/connlib/libs/common/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "libs-common"
```

Comment:
I'd suggest to break this up into more dedicated crates rather sooner than later (e.g. the `phoenix-channel`).

The problem is, "common" crates tend to become a dependency magnet. Anything that is shared between 1+ other crates easily goes into `common`. And because everything depends on it, Rust needs to rebuild everything if you change a single line in here, making iteration rather painful.

As long as we don't publish anything to crates.io and all our releases are binary releases, making new crates is cheap and moving stuff between crates is rather easy as we don't need to worry about breaking changes :)

Reply:
Taking note to do this soon, after this PR is merged